src/games/strategy/engine/data/export/GameDataExporter.java

```diff
@@ -148,21 +149,16 @@ private String technologies(final GameData data) {
   @SuppressWarnings("unchecked")
   private void propertyList(final GameData data) {
     xmlfile.append("    <propertyList>\n");
+    final GameProperties gameProperties = data.getProperties();
     try {
-      final GameProperties gameProperties = data.getProperties();
       final Field conPropField = GameProperties.class.getDeclaredField("m_constantProperties");
```

Comment:
Could you add a TODO comment indicating we have unchecked reflection here. This is something we should fix, huge liability to know that ""m_constantProperties"" refers to a variable name somewhere.